docs/reference/query-dsl/fuzzy-query.asciidoc

```diff
@@ -1,75 +1,99 @@
 [[query-dsl-fuzzy-query]]
 === Fuzzy Query
 
-The fuzzy query uses similarity based on Levenshtein edit distance.
+Returns documents that contain terms similar to the search term.
 
-==== String fields
+{es} measures similarity, or fuzziness, using a
+http://en.wikipedia.org/wiki/Levenshtein_distance[Levenshtein edit distance]. An
+edit distance is the number of one-character changes needed to turn one term
+into another. These changes can include:
 
-The `fuzzy` query generates matching terms that are within the
-maximum edit distance specified in `fuzziness` and then checks the term
-dictionary to find out which of those generated terms actually exist in the
-index. The final query uses up to `max_expansions` matching terms.
+* Changing a character (**b**ox → **f**ox)
+* Removing a character (**b**lack → lack)
+* Inserting a character (sic → sic**k**)
+* Transposing two adjacent characters (**ac**t → **ca**t)
 
-Here is a simple example:
+To find similar terms, the `fuzzy` query creates a set of all possible
+variations, or expansions, of the search term within a specified edit distance.
+The query then returns exact matches for each expansion.
 
-[source,js]
---------------------------------------------------
-GET /_search
-{
-    "query": {
-       "fuzzy" : { "user" : "ki" }
-    }
-}
---------------------------------------------------
-// CONSOLE
-
-Or with more advanced settings:
+[[fuzzy-query-ex-request]]
+==== Example request
 
 [source,js]
---------------------------------------------------
+----
 GET /_search
 {
     "query": {
-        "fuzzy" : {
-            "user" : {
+        "fuzzy": {
+            "user": {
                 "value": "ki",
-                "boost": 1.0,
-                "fuzziness": 2,
+                "fuzziness": "AUTO",
+                "max_expansions": 50,
                 "prefix_length": 0,
-                "max_expansions": 100
+                "transpositions": true,
+                "rewrite": "constant_score"
             }
         }
     }
 }
---------------------------------------------------
+----
 // CONSOLE
 
-[float]
-===== Parameters
+[[fuzzy-query-top-level-params]]
+==== Top-level parameters for `fuzzy`
+`<field>`::
+(Required, object) Field you wish to search.
 
-[horizontal]
-`fuzziness`::
+[[fuzzy-query-field-params]]
+==== Parameters for `<field>`
+`value`::
+(Required, string) Term you wish to find in the provided `<field>`.
 
-    The maximum edit distance. Defaults to `AUTO`. See <<fuzziness>>.
-
-`prefix_length`::
-
-    The number of initial characters which will not be ``fuzzified''. This
-    helps to reduce the number of terms which must be examined. Defaults
-    to `0`.
+`fuzziness`::
```

Comment:
fuzziness is used in more than one query. That's why it is good to keep its definition in a single separate documents that other queries can refer to.  It is also would be easier to change a single document if something is changed. Is there a reason to expand this definition here?

Reply:
This is a great point. I replaced this with a cross-reference in 9cec50a093e.